cli/tests/default/e2e/snapshots/test_rule_validation/test_validation_of_invalid_rules/rulesinvalid-rulesmissing-pattern.yaml/results.txt

```diff
@@ -0,0 +1,13 @@
+Configuration is invalid - found 2 configuration error(s), and 0 rule(s).
+semgrep error: Invalid rule schema
+  --> rules/invalid-rules/missing-pattern.yaml:2
+2 | - id: eqeq
+3 |   message: |
+4 |     You can't use string literals as patterns in old-style rules!
+5 |   languages: [py]
+6 |   severity: WARNING
+
+One of these properties is missing: 'match', 'pattern', 'pattern-either', 'pattern-regex', 'patterns', 'r2c-internal-project-depends-on'
```

Comment:
@akuhlens how did you get this part about r2c-internal-project-depends-on in the error message?
After merging your PR, I can't make this test pass locally, and neither in CI.
Did you forgot a PR on rule_schema_v1.jsonschema? and a link to an appropriate semgrep-interfaces commit?

Reply:
Sorry, I think this error message was recorded while I had a dirty repo state and had tried to fix the error in the wrong place, but this parse error message does make sense. I will open a PR for updating the error message so that it passes in CI and locally. Weird that it made it through the repo tests without failing, right?

Reply:
How are you seeing this failure in CI, I can't seem to find it. Did you add a commit somewhere that I didn't see to change this file. Either way, I think [this PR](https://github.com/semgrep/semgrep/pull/10217/files) addresses the issue with this snapshot and PR.